msal/src/main/java/com/microsoft/identity/client/AuthenticationResultAdapter.java

```diff
@@ -35,11 +33,13 @@
 import java.util.Locale;
 import java.util.Set;
 
-class AuthenticationResultAdapter {
+import androidx.annotation.NonNull;
+
+public class AuthenticationResultAdapter {
 
     private static final String TAG = AuthenticationResultAdapter.class.getName();
 
-    static IAuthenticationResult adapt(@NonNull final ILocalAuthenticationResult localAuthenticationResult) {
+    public static IAuthenticationResult adapt(@NonNull final ILocalAuthenticationResult localAuthenticationResult) {
```

Comment:
Why?

Reply:
We are going to use it in SignIn code from a file in different package. We need to the change access modifier from `package` to `public`.